base/components/PropControl.jsx

```diff
@@ -192,6 +194,40 @@ const PropControl = ({className, ...props}) => {
 	// Hack: Checkbox has a different html layout :( -- handled below
 	const isCheck = PropControl.KControlType.ischeckbox(type); // || PropControl.KControlType.isradio(type);
 
+	const isPublished = () => {
+
+		const getCTYPES = (typeString) => {
+			switch (typeString) {
+				case 'Campaign': return C.TYPES.Campaign;
+				case 'Charity': return C.TYPES.Charity;
+				case 'NGO': return C.TYPES.NGO;
+				case 'Advert': return C.TYPES.Advert;
+				case 'Agency': return C.TYPES.Agency;
+				case 'Advertiser': return C.TYPES.Advertiser;
+				default: return null;
+			};
+		};
+
+		if (props.item && props.path) {
+			let itemType = props.path[1];
+			let itemId = props.path[2];
+			let type = getCTYPES(itemType);
+			if (type === null) return true;
+
+			let pvDraft = getDataItem({type:type, id:itemId, status:KStatus.DRAFT, swallow:true});
+			let pvPub = getDataItem({type:type, id:itemId, status:KStatus.PUBLISHED, swallow:true});
+
+			if (pvDraft.value && pvPub.value) {
+				let valueDraft = pvDraft.value[prop];
+				let valuePub = pvPub.value[prop];
+				if (valueDraft !== valuePub) {
```

Comment:
How does this `!==` test behave with complex data-types, like `Money` which is a json object?

We might have to use `_.isEqual()` instead.